x/savings/types/expected_keepers.go

```diff
@@ -29,7 +29,7 @@ type AccountKeeper interface {
 // SavingsHooks event hooks for other keepers to run code in response to Savings modifications
 type SavingsHooks interface {
 	AfterSavingsDepositCreated(ctx sdk.Context, deposit Deposit)
-	BeforeSavingsDepositModified(ctx sdk.Context, deposit Deposit, incomingDenoms []string)
+	BeforeSavingsDepositModified(ctx sdk.Context, deposit Deposit, newDenoms []string)
```

Comment:
nit: if we removed the `Deposit` type from the hook (and just pass address and coins), it would save implementers having to import that type. Keeps modules a little bit more decoupled